docusaurus.config.js

```diff
@@ -23,23 +23,14 @@ module.exports = {
   // do not delete the following 'noIndex' line as it is modified for production
   noIndex: true,
   plugins: [
-    //        ["@edno/docusaurus2-graphql-doc-generator",
-    //          {
-    //            schema: "http://localhost:8080/tasklist/graphql",
-    //            rootPath: "./docs/", // docs will be generated under (rootPath/baseURL)
-    //            baseURL: "apis-tools/tasklist-api",
-    //            linkRoot: "/docs/",
-    //            loaders: {
-    //              UrlLoader: "@graphql-tools/url-loader"
-    //            }
-    //          },
-    //        ],
     // This custom Osano plugin must precede the gtm-plugin.
     "./static/plugins/osano",
     [
-      require.resolve("docusaurus-gtm-plugin"),
+      "./static/plugins/gtm",
       {
-        id: "GTM-KQGNSTS", // GTM Container ID
+        containerId: "GTM-KQGNSTS",
+        tagManagerUrl:
+          process.env.TAG_MANAGER_URL || "https://ssgtm.camunda.com",
```

Comment:
I'm letting the default value flow through for now; if we decide in the future that we want to change it again, we can do so by specifying an environment variable in GitHub.